src/client/Client.ts

```diff
@@ -12,33 +12,18 @@ export interface ClientOptions {
   intents?: number;
   status?: "online" | "idle" | "dnd" | "invisible";
   activityName?: string;
-  activityType: ActivityTypes;
+  activityType?: ActivityTypes;
 }
 
 export class Client extends EventEmitter {
   private socket: ws;
-
-  /**
-   * @param {ClientOptions} options passed into the client
-   */
-
   public options: ClientOptions;
-
-  /**
-   * @param {string} the bots token
-   */
-
   public token: string;
-
-  public handler: RestHandler = new RestHandler(this);
-
+  public handler: RestHandler = new RestHandler(this); 
   public manager: Manager = new Manager(this);
-
   public slashCommands: SlashCommandsManager = new SlashCommandsManager(this);
-
   public guilds: Map<string, Guild> = new Map();
-
-  public constructor(options: ClientOptions) {
+  public constructor(options?: ClientOptions) {
```

Comment:
Whats the point of all the stuff above

Reply:
above? What do you mean

Reply:
all the additions and deletions above

Reply:
They aren't needed

Reply:
why are they there then

Reply:
They were there for a reason I'm not aware of since I didn't put it there. Like I said, I removed them as there is no point adding it until we either setup typedoc properly or something else to parse it.

Reply:
i see